kit/score/score_test.go

```diff
@@ -32,3 +36,56 @@ func TestNormalize(t *testing.T) {
 		t.Errorf("Normalize(%d) = %d, expected %d", newMaxScore, sc.Score, expectedScore)
 	}
 }
+
+func TestScoreDetails(t *testing.T) {
+	sc := &score.Score{}
+
+	messages := []string{"first", "second", "third"}
+	expectedMessages := messages[:2]
+
+	wg := &sync.WaitGroup{}
+	wg.Add(1)
+	// run the test in a goroutine to avoid Fatalf
```

Comment:
This type of testing could benefit from the test framework that Russ talks about in the testing talk… the one that takes can check exit codes and output strings etc.